packages/blade/src/components/Input/BaseInput/BaseInput.tsx

```diff
@@ -143,17 +187,17 @@ const useInput = ({
 };
 
 export const getHintType = ({
-  _validationState,
+  validationState,
   hasHelpText,
 }: {
-  _validationState: BaseInputProps['validationState'];
+  validationState: BaseInputProps['validationState'];
   hasHelpText: boolean;
 }): FormHintProps['type'] => {
-  if (_validationState === 'error') {
+  if (validationState === 'error') {
     return 'error';
   }
 
-  if (_validationState === 'success') {
+  if (validationState === 'success') {
     return 'success';
   }
```

Comment:
Btw this logic is intentional right? 

Since it will fallback to helptext all the time. 

<img width="459" alt="image" src="https://user-images.githubusercontent.com/35374649/185322573-8fcae0d2-4327-48a4-90c7-e6759f63009b.png">

Reply:
🐛

Reply:
fixed in [`0606d8c` (#632)](https://github.com/razorpay/blade/pull/632/commits/0606d8cb34dc29513c054233e0d4d904ccbb9e53)